Cargo.toml

```diff
@@ -24,11 +24,10 @@ exclude = [".github/"]
 codegen-units = 1
 lto = true
 
-[profile.test-release]
-inherits = "release"
-debug = true
-debug-assertions = true
-overflow-checks = true
+[profile.test-dev]
+inherits = "dev"
+opt-level = 1
```

Comment:
I'm pleasantly surprised this was good enough :)

Reply:
Me too, I started turning off features on by one and was surprised the execution time stayed the same. What I'm not sure yet is whether this is also the optimum for CI, but I guess so.